modules/shapefile/src/lib/parsers/parse-dbf.js

```diff
@@ -15,13 +15,16 @@ const STATE = {
 class DBFParser {
   constructor({encoding}) {
     this.binaryReader = new BinaryChunkReader();
-    this.textDecoder = new TextDecoder(encoding);
+    this.textDecoder = new TextDecoder(encoding || 'latin1');
```

Comment:
I found that sometimes when `cpg` files were missing, `null` would be passed through to here, and I learned that in `const {a = 1} = {a: null}`, `null` takes preference over 1.